pom.xml

```diff
@@ -103,19 +103,6 @@
                     <autoVersionSubmodules>true</autoVersionSubmodules>
                 </configuration>
             </plugin>
-            <plugin>
```

Comment:
Why is this removed?

Reply:
it is configured inside component pom, so removed it from parent pom